src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -123,7 +126,7 @@ public Status getStatus() {
     @Override
     public void enter(Project project) {
         mainCatalogue.enter(project);
-        updateFilteredProjectList(p -> p.isSameProject(project));
```

Comment:
May you revert this line first? Because some of other code that I wrote is dependent on the filtered project list, and deleting this line may cause some bugs at this stage. I will probably deal with the problem later and delete this line together.